src/main/java/binaries/alejandro/BinaryArray.java

```diff
@@ -18,6 +18,10 @@ public BinaryArray(int[] binaries) {
      * Returns the decimal numeric value equivalent to the list of binaries stored
      */
     public int ConvertToDecimalInt() {
-        throw new UnsupportedOperationException("Not supported yet.");
+	    int result = 0;
+		for (int i=binaries.length-1; i>0; i--) {
+			result += (binaries[i])*(Math.pow( 2,binaries.length - 1 - i));
```

Comment:
Maybe you can use more meaningful variable names instead of `i`. Also, `2` and `1` look like magic numbers